src/__tests__/index.js

```diff
@@ -155,29 +155,28 @@ test('children can be an array (for preact support)', () => {
 
 test('onToggle gets called in controlled prop scenario', () => {
   const spy = jest.fn()
-  const {wrapper} = setup({on: false, onToggle: spy})
+  const {setOn, setOff, wrapper} = setup({on: false, onToggle: spy})
```

Comment:
I dont think I've edited this test when working on this PR - Ive done some branch changing, stashing etc so maybe this came from some other branch, gonna investigate it later